chunks/chunks.go

```diff
@@ -285,10 +285,13 @@ type Reader struct {
 	cs []io.Closer
 
 	pool chunkenc.Pool
+
+	// The size of the chunks
+	size int64
```

Comment:
I don't think you're actually using this anywhere

Reply:
It's being set when you create a new dir reader. ``func NewDirReader(dir string, pool chunkenc.Pool)`` is where the size is calculated and then passed along when it calls ``newReader(bs, cs, pool, sz)``